gravitee-apim-portal-webui-next/src/styles.scss

```
@@ -15,3 +15,23 @@
  */
 
 /* You can add global styles to this file, and also import other style files */
+
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@include mat.core;
```

I believe that this needs to be a function since it's a mixin.

```suggestion
@include mat.core();
```

---

Done